src/main/java/com/powsybl/openloadflow/network/LfVscConverterStation.java

```diff
@@ -14,4 +16,6 @@ public interface LfVscConverterStation extends LfGenerator {
     double getLossFactor();
 
     void setHvdc(LfHvdc hvdc);
```

Comment:
Just a detail but: here getting an hvdc could return null and getting the other bus is an optional. Maybe we have to choose one or the other for both.

Reply:
There is no getter for the HVDC at this time.

However (in the current implementation - after your review) the lfHVDC is always resent unless not present in the IDM.
But even hen present the can be an LfBus only at one end... if the other side is in a different syncrhonized network.